tests/mssql.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sql-server
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: mssql-deployment
+  namespace: sql-server
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: mssql
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: mssql
+        image: mcr.microsoft.com/mssql/server:2017-latest
```

Comment:
Ideally, for the sake of CI runtime, we want to point this to an image that has an already-instantiated database. That way, we don't have to wait the roughly 10 minutes for the create job.

I messed around with trying to commit a running pod to an image within OCP. It's theoretically possible, but it seems like the tooling isn't all in place right now. It's easy enough to locate the running pod on the OCP node with cri-o, but the `crictl` command doesn't have a `commit` equivalent to the docker command. The `podman` command _can_ do a commit, but in its current version `podman` cannot see running cri-o pods (there is a feature divergence there that they are apparently working to close).